.circleci/src/workflows/web.yml

```diff
@@ -57,6 +57,11 @@ jobs:
       requires:
         - web-build-staging
 
+  - web-test:
+      context: Audius Client
+      requires:
+        - web-build-staging
```

Comment:
could `turbo run test` instead and have the test build its deps but figure why do builds twice.